### PR TITLE
Reformatted file as per suggestions

### PR DESCRIPTION
**Pull request by Ksenia-Galtsova**

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -1,81 +1,79 @@
...
+* [Channel](channels.md#channels) list extended. The latest Channel added has code `833`.
 
 ## 26th March 2020 12:00 UTC
 
-[Amount](mews-api.md#amount) extended in [Process Group](mews-api.md#process-group) and [Price](channel-manager-api.md#price) extended in [Update Prices](channel-manager-api.md#update-prices). Net pricing is now supported for inventory upload and reservation delivery.
+* [Amount](mews-api.md#amount) extended in [Process Group](mews-api.md#process-group) and [Price](channel-manager-api.md#price) extended in [Update Prices](channel-manager-api.md#update-prices). Net pricing is now supported for inventory upload and reservation delivery.
 
 ## 19th March 2019 22:00 UTC
 
 [Channel](channels.md#channels) list extended. The latest Channel added has code `819`.
```
Comment (patrikmiksovsky95) [inline changelog.md:31]: missing here

File: changelog.md [hunk 2]
```diff
@@ -86,15 +84,15 @@ Following changes are done in a way that the previous fields still exist in both
...
 
-[Channel](channels.md#channels) list extended. The latest Channel added has code `443`.
+* [Channel](channels.md#channels) list extended. The latest Channel added has code `443`.
 
 ## 18th January 2018 00:00 UTC
 
-[Channel](channels.md#channels) list extended. The latest Channel added has code `441`.
+* [Channel](channels.md#channels) list extended. The latest Channel added has code `441`.
 
 ## 10th January 2018 22:00 UTC
```
Comment (patrikmiksovsky95) [inline changelog.md:97]: missing on line 99

File: changelog.md (hunk 1 above)
Comment (Ksenia-Galtsova) [reply changelog.md]: Done

File: changelog.md (hunk 2 above)
Comment (Ksenia-Galtsova) [reply changelog.md:94]: Done

**Review by patrikmiksovsky95 — APPROVED** [review]

**Comment by siroky** [comment]
👏

**Pull request merged**